### DISCUSSION TITLE
Nextcloud AIO 4.5.0 does not auto-update in Nextcloud 25.0.4.1

### DISCUSSION THREAD

**Discussion by tfedder-de** [nextcloud/all-in-one#2104] [Questions] [answered]
### Steps to reproduce
1. ?

### Expected behavior 
Nextcloud AIO runs backup nightly, updates Nextcloud or apps if necessary and starts it up again, all automatically.

### Actual behavior 
Nextcloud AIO run the backup this night and it started the containers. But the Nextcloud is not accessible and did not send the backup & update notifications via email. Apps can not connect to the Nextcloud.
When visited via webbrowser it states that an update is necessary, that automated updates were deactivated in config.php, and that the updater should be used via the cli.

#### Other valuable info

Nextcloud was updated to 25.0.4.1 the night before and seemed to run just fine that day.

I tried the following without success:
1. stopped the Apache container to access the Nextcloud AIO Dashboard
2. stopped all containers
3. stopped the mastercontainer through docker compose
4. updated and upgraded software packages
5. rebooted the host machine
6. started the mastercontainer through docker compose
7. started the containers via Nextcloud AIO Dashboard

I also checked the nextcloud-aio network, but it was `"Internal": false,` so it doesn't seem to be issue 2065.

**Comment by szaimen** [comment]
Hi, can you post the nextcloud container logs here?

**Comment by szaimen** [comment, marked as answer]
> When visited via webbrowser it states that an update is necessary, that automated updates were deactivated in config.php, and that the updater should be used via the cli.

I suppose you are running into this https://github.com/nextcloud/all-in-one/discussions/1787

**Comment by tfedder-de** [comment]
Hi @szaimen, 

thank you very much for your quick help.
Yes, I was running into #1787. The solution you described there fixed it for now. I am sorry, that I did not find that issue on my own. The culprit was the app Polls updating to 4.1.7.

Are the logs of the Nextcloud container of any use to you beyond this?

**Reply by szaimen** [reply]
> Are the logs of the Nextcloud container of any use to you beyond this?

Not really. If you found the app responsible for this, feel free to create an issue in their repository.
